packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -587,8 +582,8 @@ async function packageResources(context, resources) {
   return Promise.all(promises);
 }
 
-async function updateCloudFormationNestedStack(context, nestedStack, resourcesToBeCreated, resourcesToBeUpdated) {
-  const backEndDir = context.amplify.pathManager.getBackendDirPath();
+async function updateCloudFormationNestedStack(context: $TSContext, nestedStack, resourcesToBeCreated, resourcesToBeUpdated) {
```

Comment:
can types be added for the other params (even $TSAny would be better than no type)